### PR TITLE
fix(Event/Midsummer): miscellaneous cleanups of spawns

### PR DESCRIPTION
**Pull request by sudlud**
## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

## Issues Addressed:

- Finishs some TODOs of #17549 

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [ ] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [ ] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1702120071739163800.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+-- 16781, 'Midsummer Celebrant'
+-- update orientations (completes TODO of PR #17549)
+UPDATE `creature` SET `orientation` = 5.19 WHERE (`id1` = 16781) AND (guid = 139128);
+UPDATE `creature` SET `orientation` = 2.86 WHERE (`id1` = 16781) AND (guid = 139129);
+UPDATE `creature` SET `orientation` = 5.94 WHERE (`id1` = 16781) AND (guid = 139130);
+UPDATE `creature` SET `orientation` = 0.84 WHERE (`id1` = 16781) AND (guid = 139151);
+UPDATE `creature` SET `orientation` = 0.73 WHERE (`id1` = 16781) AND (guid = 139153);
+UPDATE `creature` SET `orientation` = 0.89 WHERE (`id1` = 16781) AND (guid = 139184);
+UPDATE `creature` SET `Comment` = "orientation corrected manually (not sniffed)" WHERE (guid IN (139128, 139129, 139130, 139151, 139153, 139184));
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1702120071739163800.sql:30]: Ticks on guid

Comment (sudlud) [reply data/sql/updates/pending_db_world/rev_1702120071739163800.sql:30]: Oh I'll re-check

File: data/sql/updates/pending_db_world/rev_1702120071739163800.sql [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+-- 16781, 'Midsummer Celebrant'
+-- update orientations (completes TODO of PR #17549)
+UPDATE `creature` SET `orientation` = 5.19 WHERE (`id1` = 16781) AND (guid = 139128);
+UPDATE `creature` SET `orientation` = 2.86 WHERE (`id1` = 16781) AND (guid = 139129);
+UPDATE `creature` SET `orientation` = 5.94 WHERE (`id1` = 16781) AND (guid = 139130);
+UPDATE `creature` SET `orientation` = 0.84 WHERE (`id1` = 16781) AND (guid = 139151);
+UPDATE `creature` SET `orientation` = 0.73 WHERE (`id1` = 16781) AND (guid = 139153);
+UPDATE `creature` SET `orientation` = 0.89 WHERE (`id1` = 16781) AND (guid = 139184);
+UPDATE `creature` SET `Comment` = "orientation corrected manually (not sniffed)" WHERE (`id1` = 16781) AND (guid IN (139128, 139129, 139130, 139151, 139153, 139184));
```
Comment (heyitsbench) [inline data/sql/updates/pending_db_world/rev_1702120071739163800.sql:30]:
```suggestion
UPDATE `creature` SET `Comment` = "orientation corrected manually (not sniffed)" WHERE (`id1` = 16781) AND (`guid` IN (139128, 139129, 139130, 139151, 139153, 139184));
```
Affects the lines above as well.

Comment (sudlud) [reply data/sql/updates/pending_db_world/rev_1702120071739163800.sql:30]: True, I'll update later today

Comment (sudlud) [reply data/sql/updates/pending_db_world/rev_1702120071739163800.sql:30]: Same issue that @kitzunu mentioned but I just didn't understand what he meant 🙈

**Review by Nyeriah — APPROVED** [review]
